src/components/Tabs/TabLabel.tsx

```diff
@@ -0,0 +1,99 @@
+import styled, { css } from 'styled-components';
+
+import {
+  getColor,
+  getFontSize,
+  getLineHeight,
+  pxToRem,
+} from '../../utils/helpers';
+import { TabSizes, TabVariants } from './Tabs.enums';
+import { LabelProps, Sizes, Variants } from './Tabs.types';
+
+const largeTextSize = css`
+  font-size: ${getFontSize('lg')};
+  line-height: ${getLineHeight('lg')};
+`;
+
+const mediumTextSize = css`
+  font-size: ${getFontSize('md')};
+  line-height: ${getLineHeight('md')};
+`;
+
+const largeUnderlineSize = css`
+  font-size: ${getFontSize('h2')};
+  line-height: ${getLineHeight('xxl')};
+`;
+
+const mediumUnderlineSize = css`
+  font-size: ${getFontSize('h3')};
+  line-height: ${getLineHeight('xl')};
+`;
+
+const smallUnderlineSize = css`
+  font-size: ${getFontSize('h4')};
+  line-height: ${getLineHeight('lg')};
+`;
+
+const underlineSizes = {
+  [TabSizes.lg]: largeUnderlineSize,
+  [TabSizes.md]: mediumUnderlineSize,
+  [TabSizes.sm]: smallUnderlineSize,
+};
+
+const textSizes = {
+  [TabSizes.lg]: largeTextSize,
+  [TabSizes.md]: mediumTextSize,
+};
+
+const underlineTab = css<LabelProps & { size: Sizes; variant: Variants }>`
+  ${({ size }) => underlineSizes[size] || underlineSizes[TabSizes.md]};
```

Comment:
The default seems way too big, maybe TabSizes.sm?
![image](https://user-images.githubusercontent.com/8337116/106865194-229c6080-66cb-11eb-8c8e-69f6c1723ee1.png)